### PR TITLE
chore(api/logging) : added "How to get logger" in logging page

### PR DESCRIPTION
**Pull request by anikethsaha**
Added the missing `how to get the logger?` content in the `api/logging` page

### CODE REVIEW

File: src/content/api/logging.md [hunk 1]
```diff
@@ -25,18 +25,52 @@ By introducing webpack logging API we hope to unify the way webpack plugins and
...
 W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose logging level as low as possible to keep the log output informative.
 
+__How to get the logger ?__
+
+- If you want to store the logs in the Stats, you should use `compilation.getLogger('PluginName')`
+
+  ```js
+  // example
+
+  apply(compiler){
```
Comment (jamesgeorge007) [inline src/content/api/logging.md]:
```suggestion
  apply(compiler) {
```

File: src/content/api/logging.md [hunk 2]
```diff
@@ -25,18 +25,52 @@ By introducing webpack logging API we hope to unify the way webpack plugins and
...
+  }
+
+  ```
+
+- If you dont want to store your logs in the Stats, use `infrastructure` logging
+
+  ```js
+  // example
+
+  apply(compiler){
```
Comment (jamesgeorge007) [inline src/content/api/logging.md]:
```suggestion
  apply(compiler) {
```

File: src/content/api/logging.md [hunk 3]
```diff
@@ -25,18 +25,52 @@ By introducing webpack logging API we hope to unify the way webpack plugins and
...
+    const logger = compiler.getInfrastructureLogger('PluginName');
+    ...
+  }
+  ```
+
+- if you want to get the logger from webpack as a node api, use
```
Comment (jamesgeorge007) [inline src/content/api/logging.md]:
```suggestion
- if you want to get the logger from webpack as a Node.js API, use
```

**Comment by anikethsaha** [comment]
cc  @montogeek @EugeneHlushko

**Review by wizardofhogwarts — DISMISSED** [review]

**Comment by anikethsaha** [comment]
@montogeek can you review this

**Comment by snitin315** [comment]
It's fixed in master. https://webpack.js.org/api/logging/#examples-of-how-to-get-and-use-webpack-logger-in-loaders-and-plugins

**Pull request closed without merging**